pkg/ast/spl/tests/splParser_test.go

```diff
@@ -8240,3 +8242,116 @@ func performEventCountTest(t *testing.T, query string, expectedIndices []string,
 	assert.Equal(t, expectedReportSize, aggregator.OutputTransforms.LetColumns.EventCountRequest.ReportSize)
 	assert.Equal(t, expectedListVix, aggregator.OutputTransforms.LetColumns.EventCountRequest.ListVix)
 }
+
+func compareStringSlices(a []string, b []string) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	for i := range a {
+		if a[i] != b[i] {
+			return false
+		}
+	}
+	return true
+}
```

Comment:
Assuming there isn't a function to do this in a library we're using already, this function probably belongs in sliceutils.go. If you want, you could even make this a generic function so that we could compare any two slices containing `Comparable` types so that we could reuse it e.g., to compare slices of ints; but you don't have to write the generic function

Reply:
addressed in latest commit